resources/js/components/Form/registration-form.js

```diff
@@ -61,23 +62,15 @@ export default class RegistrationForm extends Component {
             </RadioGroup>
           </div>
           <div class="col">
-            {this.state.AccountType == "Organization" ? (
```

Comment:
I am going to add this conditional back in so that this input is only visible if the user has selected to register an organization

Reply:
I'm not yet sure how to handle adding a user to an organization if this input is not filled. Perhaps there should be a 'default' organization that every normal user is added to.